pkg/fqdn/dnsproxy/proxy.go

```diff
@@ -384,6 +384,15 @@ func (e ErrTimedOutAcquireSemaphore) Error() string {
 	)
 }
 
+// ErrDNSRequestNoEndpoint represents an error when the local daemon cannot
```

Comment:
>when the local daemon cannot

Do you mean the dns proxy?

Reply:
Technically, the daemon has the source of truth for endpoints.